install-php-locally.md

```diff
@@ -0,0 +1,177 @@
+
+```terminal
+brew install symfony-cli/tap/symfony-cli
+```
+
+You should be ready to go on your Mac! [Verify the installation below](#verify-phpsymfony-cli-installation).
+
+## Ubuntu/Debian Linux (apt)
+
+In Ubuntu and Debian, you can install PHP using the `apt` package manager but the default package repository probably
+won't have the latest PHP version available. The 3rd-party [`ondrej/php` repository](https://launchpad.net/~ondrej/+archive/ubuntu/php)
+is a popular choice that provides the latest PHP versions and extensions. 
+
+### APT PHP
+
+First, add the `ondrej/php` repository to your system by running:
+
+```terminal
+sudo add-apt-repository ppa:ondrej/php
+sudo apt update
+```
+
+To install PHP 8.4 and some common extensions, run:
+
+```terminal
+sudo apt install php8.4 php8.4-cli php8.4-common php8.4-curl php8.4-mbstring php8.4-xml php8.4-zip php8.4-gd php8.4-intl php8.4-bcmath 
+```
+
+To install the common database extensions, run:
+
+```terminal
+sudo apt install php8.4-mysql php8.4-pgsql php8.4-sqlite3
+```
+
+***NOTE
+Replace `8.4` with the version you want to install. For example: `php8.3 php8.3-cli ...` for PHP 8.3.
+***
+
+### APT PHP Extensions
+
+For additional PHP extensions, these are also available via `apt`. For example, to install
+the Redis extension, run:
+
+```terminal
+sudo apt install php8.4-redis
+```
+
+### APT Symfony CLI
+
+There are several methods to [install the Symfony CLI](https://symfony.com/download), since we used
+`apt` to install PHP, let's use it to install the Symfony CLI as well:
+
+```terminal
+curl -1sLf 'https://dl.cloudsmith.io/public/symfony/stable/setup.deb.sh' | sudo -E bash
+sudo apt install symfony-cli
+```
+
+You should be ready to go! [Verify the installation below](#verify-phpsymfony-cli-installation).
+
+## Windows (WSL)
+
+Running PHP natively on Windows (as an `exe`) is *possible*, but can be tricky. I recommend using the
+"Windows Subsystem for Linux" (WSL) to run Ubuntu. It isn't in a container or anything, it runs
+*directly* on Windows! Follow the [WSL installation guide](https://learn.microsoft.com/en-us/windows/wsl/install)
+to get setup with WSL and Ubuntu.
+
+Now, follow the [Ubuntu/Debian Linux (apt)](#ubuntudebian-linux-apt) section above to get PHP installed!
+
+## Verify PHP/Symfony CLI Installation
+
+Check your PHP version by running:
+
+```terminal
+php -v
+```
+
+Check your Symfony CLI version by running:
+
+```terminal
+symfony version
+```
+
+You can run PHP *through* the Symfony CLI. Ensure the following matches your PHP version above:
+
+```terminal
+symfony php -v
+```
+
+For the common database extensions, ensure they are all installed by running:
+
+```terminal
+symfony php -m | grep pdo
+```
+
+You should see the following output:
+
+```
+pdo_mysql # MySQL extension
+pdo_pgsql # PostgreSQL extension
+pdo_sqlite # SQLite extension
+```
+
+Finally, to ensure that you're ready to go with Symfony, run:
+
+```terminal
+symfony check:requirements
+```
+
```

Comment:
How about adding a note that you can use this command in the root directory of your Symfony project to check project-specified requirements?